openml/datasets/functions.py

```diff
@@ -1061,6 +1061,61 @@ def fork_dataset(data_id: int) -> int:
     return int(data_id)
 
 
+def data_feature_add_ontology(data_id, index, ontology):
+    """
+    Adds an ontology (URL) to a given dataset feature (defined by a dataset id
+    and index). The dataset has to exists on OpenML and needs to have been
+    processed by the evaluation engine.
+
+    Parameters
+    ----------
+    data_id : int
+        id of the dataset to which the feature belongs
+
+    index : int
+        index of the feature in dataset (0-based)
```

Comment:
Which list is this based on? I.e., which 0-based list of a dataset object is used? It might help the user to orientate with the dataset.